airgun/views/computeresource.py

```diff
@@ -174,6 +178,8 @@ class virtual_machines(SatTab, SearchableViewMixin):
                 'Power': Text('.//span[contains(@class,"label")]'),
             }
         )
+        more_button = Text('//span[@class="caret"]')
+        import_button = Text('//a[text()="Import"]')
```

Comment:
One more question:
Are you sure that all compute providers ('ec2', 'docker', 'rhev' and etc) have these elements? Otherwise that code will add more regression than value